### PR TITLE
Zone.strong

### PR DESCRIPTION
**Pull request by floitschG**
The `dart:async` zone API is in the process of becoming strong-mode clean.
This patch fixes a breakage introduced by this change.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+version: 1.3.3
 author: Dart Team <misc@dartlang.org>
 description: A class for managing a finite pool of resources.
 homepage: https://github.com/dart-lang/pool
 dependencies:
   async: "^1.4.0"
   stack_trace: ">=0.9.2 <2.0.0"
 environment:
-  sdk: ">=1.22.0 <2.0.0"
+  sdk: ">=2.0.0-dev.0.1 <2.0.0-dev.infinity"
```
Comment (kevmoo) [inline pubspec.yaml]: 2 things

1) This won't be a . release. It'll be dev.1.0 – I think
2) Don't need the infinity thing any more. Just <2.0.0 will work...

Comment (kevmoo) [reply pubspec.yaml]: ...also, update changelog!

**Comment by floitschG** [comment]
done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
-## 1.3.1
```
Comment (kevmoo) [inline CHANGELOG.md]: Don't delete the 1.3.1 entry!

File: pubspec.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: pool
-version: 1.3.1
+version: 1.3.3
```
Comment (kevmoo) [inline pubspec.yaml]: 1.3.2?

File: CHANGELOG.md (hunk 2 above)
Comment (floitschG) [reply CHANGELOG.md]: done.

File: pubspec.yaml (hunk 3 above)
Comment (floitschG) [reply pubspec.yaml]: done.

**Comment by floitschG** [comment]
done.

File: lib/pool.dart [hunk 4]
```diff
@@ -183,8 +183,7 @@ class Pool {
       _allocatedResources--;
       if (_allocatedResources == 0) _closeGroup.close();
     } else {
-      _onReleaseCallbacks.add(
-          Zone.current.bindCallback(onRelease, runGuarded: false));
+      _onReleaseCallbacks.add(Zone.current.bindCallback(onRelease));
```
Comment (kevmoo) [inline lib/pool.dart]: To be super clear – we will now run these callbacks "guarded" in 1.24?

**Pull request merged**